### DISCUSSION TITLE
Policies

### DISCUSSION THREAD

**Discussion by notramo** [martenframework/marten#94] [Ideas]
Please add policies, similar to https://github.com/varvet/pundit

**Comment by ellmetha** [comment]
What would be the rationale for this being provided by the framework (instead of being provided as a separate application/shard)? I am not familiar with the gem you linked, and from what I can see it looks like a nice way to define authorization-related logic. I am not entirely certain if this feature is significant enough to be included as a core feature in the framework. However, I do think it could be offered as a standalone application or shard!
